tests/Struct/FindologicProductTest.php

```diff
@@ -363,9 +364,15 @@ public function testProductPrices(?Price $price, ?string $exception): void
 
     public function testProduct(): void
     {
-        $productEntity = $this->createTestProduct();
+        $productId = '29d554327a16fd51350688cfa9930b29';
+        $languageId = Defaults::LANGUAGE_SYSTEM;
+        $productEntity = $this->createTestProduct([
+            'searchKeywords' => [
+                ['id' => Uuid::randomHex(), 'productId' => $productId, 'languageId' => $languageId, 'keyword' => 'FINDOLOGIC Keyword', 'ranking' => 500]
```

Comment:
Run `composer lint` to ensure there are no errors before pushing. For e.g. in this line it seems there are more than allowed characters in a single line. So please fix that.